src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	
+	private String status;
+	public Tagging(Tag tag, Person person, boolean add) {
+		if (add) {
```

Comment:
As discussed in class, Tagging is for the relation between Person and Tag so Person and Tag should be stored instead of a String. Remember to follow coding standard for naming (isAdd instead of add)